igor-web/src/main/java/com/netflix/spinnaker/igor/scm/gitlab/client/model/CompareCommitsResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 bol.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm.gitlab.client.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CompareCommitsResponse {
+    public final List<Commit> commits;
+
+    @JsonCreator
+    public CompareCommitsResponse(@JsonProperty("commits") List<Commit> commits) {
```

Comment:
You can remove the annotation value

Reply:
Can't, Jackson gets confused if I do

Reply:
Yeah, you are correct. The JavaDoc on @JsonProperty seems to suggest that it's not needed, but the documentation on @JsonCreator is quite explicit on it being needed in plain Java code.